### PR TITLE
refactor(screen.c): resolve all clint errors

### PR DESCRIPTION
**Pull request by lewis6991**

### CODE REVIEW

File: src/nvim/screen.c [hunk 1]
```diff
@@ -6453,18 +6346,14 @@ static int fillchar_hsep(win_T *wp, int *attr)
   return wp->w_p_fcs_chars.horiz;
 }
 
-/*
- * Return TRUE if redrawing should currently be done.
- */
+/// Return TRUE if redrawing should currently be done.
 int redrawing(void)
```
Comment (zeertzjq) [inline src/nvim/screen.c:6350]:
```suggestion
/// Return true if redrawing should currently be done.
bool redrawing(void)
```

File: src/nvim/screen.c [hunk 2]
```diff
@@ -6453,18 +6346,14 @@ static int fillchar_hsep(win_T *wp, int *attr)
...
 {
   return !RedrawingDisabled
          && !(p_lz && char_avail() && !KeyTyped && !do_redraw);
 }
 
-/*
- * Return TRUE if printing messages should currently be done.
- */
+/// Return TRUE if printing messages should currently be done.
 int messaging(void)
```
Comment (zeertzjq) [inline src/nvim/screen.c:6357]:
```suggestion
/// Return true if printing messages should currently be done.
bool messaging(void)
```

**Review by dundargoc — APPROVED** [review]

**Review by bfredl — APPROVED** [review]

**Comment by clason** [comment]
Uncrustify still complains:
```diff
@@ -6484,7 +6484,8 @@ static void win_redr_ruler(win_T *wp, bool always)
     // Some sprintfs return the length, some return a pointer.
     // To avoid portability problems we use strlen() here.
     vim_snprintf(buffer, RULER_BUF_LEN, "%" PRId64 ",",
-                 (wp->w_buffer->b_ml.ml_flags & ML_EMPTY) ? (int64_t)0L : (int64_t)wp->w_cursor.lnum);
+                 (wp->w_buffer->b_ml.ml_flags &
+                  ML_EMPTY) ? (int64_t)0L : (int64_t)wp->w_cursor.lnum);
     size_t len = STRLEN(buffer);
     col_print(buffer + len, RULER_BUF_LEN - len,
               empty_line ? 0 : (int)wp->w_cursor.col + 1,
```

**Review by justinmk — APPROVED** [review]
nice!

**Pull request merged**